samples/drivers/led_lp503x/boards/lpcxpresso11u68.overlay

```diff
@@ -9,7 +9,7 @@
 &i2c0 {
 	/* TI LP5030 LED controller connected to I2C0. */
 	lp5030@30 {
-		compatible = "ti,lp5030", "ti,lp503x";
+		compatible = "ti,lp5030", "ti,lp50xx";
```

Comment:
I think this patch should be squashed with `drivers: led: lp503x: extend driver with support for LP5009/LP5012`.

Else this sample is broken between the two patches, and this can harm `git-bisect`.

Moreover it doen't build. You also need to fix the following line in `samples/drivers/led_lp503x/src/main.c`:

`const struct device *lp503x_dev = DEVICE_DT_GET_ANY(ti_lp503x);`

should be:

`const struct device *lp503x_dev = DEVICE_DT_GET_ANY(ti_lp50xx);`

In addition the sample name, some variables names and the README file should be updated as well: lp503x -> lp50xx.